### DISCUSSION TITLE
OpenGL plays better than CudaNN

### DISCUSSION THREAD

**Discussion by JohnDijkstra** [LeelaChessZero/lc0#1691] [General]
I have been playing several matches of LC0 (0.27.0) using weights file T40B.4-160 (the latest official 20x256 weights file for the GTX 150) on a Q6600 w/ GTX1050ti platform against Stockfish 10. When I play the CudaNN version, this is barely able to get an advantage over SF10 and when it does, it does the usual beating around the bush in the end game until a drawish position is achieved. Surprisingly -at least to me- when I play the same weights file but then the OpenGL version, LC0 from the start of the game has a clear advantage over SF10 and almost every time wins. The search depth and NPS are much less of course then the CudaNN version. Overall, the playing style is more tactical throughout the game and the end game is of a much higher level, more tactical in nature where LC0 is working towards a clear win.

Any thoughts on this surprising difference?

**Comment by mooskagh** [comment]
It looks to me that it's more just a factor of chance (and subjective judgement of a play style), rather than real effect.
I'm pretty sure that if you computed some statistics methodically, the result would be higher NPS -> stronger play (given the same network and time control).
